### PR TITLE
Extending AccountsApi with SSH key functionality

### PR DESCRIPTION
**Pull request by PMooreWand**
Adding SSH Key addition, list and deletion to AccountsApi

### CODE REVIEW

File: src/main/java/com/urswolfer/gerrit/client/rest/http/accounts/SshKeysParser.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.urswolfer.gerrit.client.rest.http.accounts;
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/accounts/SshKeysParser.java:1]: Can you please add a licence header as present in the other files?

File: src/main/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountApiRestClient.java [hunk 2]
```diff
@@ -83,4 +90,24 @@ public BinaryResult downloadAvatar(int size) throws RestApiException {
...
+    @Override
+    public List<SshKeyInfo> listSshKeys() throws RestApiException {
+        String request = getRequestPath() + "/sshkeys";
+        JsonElement result = gerritRestClient.getRequest(request);
+        return sshKeysParser.parseSshKeyInfos(result);
+    }
+
+    @Override
+    public void deleteSshKey(int seq) throws RestApiException {
+        String request = getRequestPath()+ "/sshkeys/"+ seq;
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountApiRestClient.java:110]: please reformat this line

File: src/test/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountsRestClientTest.java [hunk 3]
```diff
@@ -43,8 +46,9 @@ public void testId() throws Exception {
     public void testSelf() throws Exception {
         GerritRestClient gerritRestClient = gerritRestClientExpectGet("/accounts/self");
         AccountsParser accountsParser = getAccountsParser();
+        SshKeysParser sshKeysParser = getSshKeysParser();
 
-        AccountsRestClient accountsRestClient = new AccountsRestClient(gerritRestClient, accountsParser);
+        AccountsRestClient accountsRestClient = new AccountsRestClient(gerritRestClient, accountsParser,sshKeysParser);
```
Comment (uwolfer) [inline src/test/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountsRestClientTest.java:51]: reformat

File: src/test/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountsRestClientTest.java [hunk 4]
```diff
@@ -28,12 +29,14 @@
...
     private static final AccountInfo MOCK_ACCOUNT_INFO = EasyMock.createMock(AccountInfo.class);
+    private static final SshKeyInfo MOCK_SSHKEY_INFO = EasyMock.createMock(SshKeyInfo.class);
 
     @Test
     public void testId() throws Exception {
         GerritRestClient gerritRestClient = gerritRestClientExpectGet("/accounts/jdoe");
         AccountsParser accountsParser = getAccountsParser();
-        AccountsRestClient accountsRestClient = new AccountsRestClient(gerritRestClient, accountsParser);
+        SshKeysParser sshKeysParser = getSshKeysParser();
+        AccountsRestClient accountsRestClient = new AccountsRestClient(gerritRestClient, accountsParser,sshKeysParser);
```
Comment (uwolfer) [inline src/test/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountsRestClientTest.java:39]: reformat

File: src/test/java/com/urswolfer/gerrit/client/rest/http/accounts/SshKeyParserTest.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+import com.google.gson.JsonElement;
+import com.urswolfer.gerrit.client.rest.http.common.AbstractParserTest;
+import com.urswolfer.gerrit.client.rest.http.common.GerritAssert;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Thomas Forrer
```
Comment (uwolfer) [inline src/test/java/com/urswolfer/gerrit/client/rest/http/accounts/SshKeyParserTest.java:31]: you can add your name here

**Comment by PMooreWand** [comment]
Pushed new commit with the requested changes.
Apologies for any updates around the force push, I'm used to Gerrit flow of single commit for reviews. If prefer this squashed let me know.

**Comment by uwolfer** [comment]
> Pushed new commit with the requested changes.

Thanks!

> Apologies for any updates around the force push, I'm used to Gerrit flow of single commit for reviews. If prefer this squashed let me know.

No worries, I've used GitHub squash functionality to merge this PR. Thanks again for your work.

**Comment by uwolfer** [comment]
If you would like to have a new release of this lib, please let me know.

**Comment by PMooreWand** [comment]
A new release would be appreciated, but its not critical. Thanks.

**Comment by uwolfer** [comment]
Released v0.9.1.

**Pull request merged**